src/scripts/main.js

```diff
@@ -1,3 +1,21 @@
 'use strict';
 
 // write code here
+document.querySelectorAll('.tree li').forEach((li) => {
+  const span = document.createElement('span');
+
+  span.textContent = li.firstChild.textContent.trim();
+
+  li.replaceChild(span, li.firstChild);
```

Comment:
Ensure that `li.firstChild` is indeed a text node. If the list item contains other elements or text nodes, `li.firstChild` might not be what you expect. Consider iterating over `li.childNodes` to find and replace only text nodes.